doc/user/content/sql/create-sink/kafka.md

```diff
@@ -54,6 +57,7 @@ _item&lowbar;name_ | The name of the source, table or materialized view you want
 **NOT ENFORCED** | Whether to disable validation of key uniqueness when using the upsert envelope. See [Upsert key selection](#upsert-key-selection) for details.
 **ENVELOPE DEBEZIUM** | The generated schemas have a [Debezium-style diff envelope](#debezium-envelope) to capture changes in the input view or source.
 **ENVELOPE UPSERT** | The sink emits data with [upsert semantics](#upsert-envelope).
+_retention_period_ | ***Private preview.** This option has known performance or stability issues and is under active development.* Duration for which Materialize retains historical data for performing [time travel queries](/transform-data/patterns/time-travel-queries)). Accepts positive [interval](/sql/types/interval/) values (e.g. `'1hr'`). Default: `1s`.
```

Comment:
Should this be here on the CREATE SINK page?

Reply:
oops, I noticed it was missing it on the create-source kafka page earlier, and now I know why! nope...will remove.